### PR TITLE
fix: ov enrolment styling update

### PR DESCRIPTION
**Pull request by shuowu-okta**
## PR Checklist

- [ ] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: assets/sass/v2/_okta-verify.scss [hunk 1]
```diff
@@ -34,6 +34,10 @@
   .sameDevice-setup {
     text-align: center;
   }
+  
+  .sameDevice-setup.ov-enrolment-enabled {
```
Comment (lesterchoi-okta) [inline assets/sass/v2/_okta-verify.scss:38]: nit: typo?

```suggestion
  .sameDevice-setup.ov-enrollment-enabled {
```

File: src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js [hunk 2]
```diff
@@ -30,16 +30,21 @@ export default View.extend({
         </ul>
       {{/if}}
       {{#if deviceMap.setupOVUrl}}
-        <div class="sameDevice-setup">        
+        <div class="sameDevice-setup{{#if sameDeviceOVEnrollmentEnabled}} ov-enrolment-enabled{{/if}}">
```
Comment (lesterchoi-okta) [inline src/v2/view-builder/views/ov/EnrollChannelPollDescriptionView.js:33]:
```suggestion
        <div class="sameDevice-setup{{#if sameDeviceOVEnrollmentEnabled}} ov-enrollment-enabled{{/if}}">        
```

File: test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js [hunk 3]
```diff
@@ -1060,7 +1061,7 @@ test
     }
   });
 
-test
+test.only
```
Comment (lesterchoi-okta) [inline test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js]:
```suggestion
test
```

File: test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js [hunk 4]
```diff
@@ -1085,14 +1086,15 @@ test
...
+      await t.expect(await enrollOktaVerifyPage.getDesktopNoPromptText()).eql(sameDeviceOVEnrollmentDesktopInstructions4);
       const downloadInstruction = await enrollOktaVerifyPage.getSameDeviceDownloadText();
       await t.expect(downloadInstruction).contains('Don’t have Okta Verify?');
       await t.expect(downloadInstruction).contains('Download here');
       await t.expect(await enrollOktaVerifyPage.appStoreElementExists()).eql(false);
     }
   });
 
-test
+test.only
```
Comment (lesterchoi-okta) [inline test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js]:
```suggestion
test
```

**Review by lesterchoi-okta — COMMENTED** [review]
Are the `test.only` intentional? Will approve if so. Typo can wait, no block

**Review by lesterchoi-okta — APPROVED** [review]

File: assets/sass/v2/_okta-verify.scss (hunk 1 above)
Comment (shuowu-okta) [reply assets/sass/v2/_okta-verify.scss:38]: the vscode showed me `enrollment ` as typo .... will change to `enrollment ` to keep consistency with other places

**Pull request merged**